tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -97,6 +97,7 @@ sub run_test {
         #check the networking of the plugged interface
         #use br123 as ssh connection
         test_network_interface($guest, gate => $gateway, mac => $vfs[0]->{vm_mac}, net => 'br123');
+        check_guest($guest);
```

Comment:
Is this the optimal place in which check_guest is put ?

Reply:
In my sriov test, the coredump failure in guest happens when the interface is brought up in test_network_interface, so I add the check here. In other cases, coredump may happend anywhere. More checks are better, but it is realy too wordy.

Reply:
There are two parts of the PR, one is to discover unsevere coredump/calltrace by check_host/guest(), the other is to try to reconnect ssh console in post_fail_hook().

check_host/guest() has nothing to do with crash and can be done anytime if you like in theory. But we do need put them in proper places with team discussion.

Reply:
I think post_fail_hook one should be implemented in more general base class module or at backend instead of here in our feature test module.

Reply:
I think post_fail_hook one should be implemented in more general base class module or at backend instead of here in our feature test module.